docs/configuration/entrypoints.md

```diff
@@ -34,6 +34,9 @@ To redirect an http entrypoint to an https entrypoint (with SNI support).
       KeyFile = "integration/fixtures/https/snitest.org.key"
 ```
 
+!!! note
```

Comment:
Instead of `!!!` can we use bolding or some sort of markdown based notification ;)

Reply:
We use `!!!` as Markdown syntax, it's related to our new documentation theme : http://squidfunk.github.io/mkdocs-material/extensions/admonition/

Reply:
So sorry about that, didn't know!